src/passes/function_passes/LiveVariablesPass.cpp

```diff
@@ -90,8 +96,7 @@ struct LiveVariablesPass: public FunctionPass {
             // `instr`, so it is live across `instr`.
             if (isa<Instruction>(*user)) {
               auto userInst = cast<Instruction>(*user);
-              if (DT.dominates(&instr, userInst) ||
```

Comment:
Can you explain this part?

Reply:
@vext01 `DT.dominates(&instr, userInst)` checks if `instr` dominates `userInst`. In this case, `instr` is the call instruction for which I want to find the live variables. Essentially, for each instruction (indicated by iterator `it`) that comes before `instr` (`!it->use_empty() && DT.dominates(&*it, &instr)`), find out which of its uses are 'below' `instr`  (`DT.dominates(&instr, userInst)`)

 I removed `instr.isSameOperationAs(&*userInst))` from the condition, because it was incorrect, as It was causing the return value of the call itself to be recorded in the stack map.